bench/BenchIndices.hs

```diff
@@ -0,0 +1,135 @@
+{-# LANGUAGE BangPatterns        #-}
+-- |
+-- Copyright   : (c) 2011 Simon Meier
```

Comment:
I guess this is worth updating. Is there any Simon's code in this file?

I'd skip `Maintainer`, `Stability` and `Portability` fields at all, this is an auxiliary file. Or you can file yourself as a maintainer of this module, if you wish.

Reply:
Aside from mimicking the layout and copying the imports of `BenchAll.hs`, all of the code in this module was written by hand. How much of the haddock header should I keep if I would already be removing `Maintainer`, `Stability`, and `Portability`?

Reply:
`Copyright` (presumably, `(c) 2020 Peter Duchovni` or whatever) and `License` should be kept.

Reply:
Done.